addons/html_editor/static/src/main/link/link_plugin.js

```diff
@@ -189,6 +189,9 @@ export class LinkPlugin extends Plugin {
         this.overlay = this.dependencies.overlay.createOverlay(LinkPopover, {}, { sequence: 50 });
         this.addDomListener(this.editable, "click", (ev) => {
             if (ev.target.tagName === "A" && ev.target.isContentEditable) {
+                if (ev.ctrlKey || ev.metaKey) {
+                    window.open(ev.target.href, "_blank");
+                }
```

Comment:
Since both the issues and their respective fixes are completely unrelated from each other, they should be fixed in separated commits. It is ok to keep them in a single PR.

In the commit message for the `_blank` fix, it is better to link to the actual commit introducing the fix rather than on an arbitrary line of code in 17.0.